Persephone/Controllers/WindowController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  WindowController.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/1/11.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+import mpdclient
+
+class WindowController: NSWindowController {
+  enum TransportAction: Int {
+    case prevTrack = 0
+    case playPause = 1
+    case stop = 2
+    case nextTrack = 3
+  }
+
+  override func windowDidLoad() {
+    super.windowDidLoad()
+    window?.titleVisibility = .hidden
+
+    NotificationCenter.default.addObserver(
```

Comment:
Does this observer need to be removed at some point in the app's lifecycle?

Reply:
Looks like this isn't necessary anymore?

https://developer.apple.com/documentation/foundation/notificationcenter/1413994-removeobserver